com.unity.render-pipelines.high-definition/Documentation~/Decal-Shader.md

```diff
@@ -4,33 +4,44 @@
 
 ## Properties
 
+### Surface Options
+
+These properties allow you to set the affected attributes of the Material the decal is project onto when HDRP renders it into the Scene.
```

Comment:
These properties allow you to set which Material attributes the decal affects when HDRP renders it into the Scene.